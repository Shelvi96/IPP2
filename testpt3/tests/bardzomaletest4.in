NEW ID

07:4:>;7
5400671;5>032
20370>57409
::0:89609>0
35>495:
652:>8
:82;;031>075
755>28;36170
12018;22:>756;514;01
74567;7>3685:15
38;47826>73584:783
8>169766:
0;;0;>77:516872
9110>1:5874
12:52:956>4
@:::943665:168:2739097
@3930749067284
@029542301:;87;9691455;92331:;9
@3;:0
@04;16:11
@;;74309714:450871
@:;46405564
@;6::5266;1;73854409;2938136:1:
@0873119;12;0575139;200
@8
@123345704:66
@5;14;;3906631:7;;00187:172558;
@22::
@923209910
@0;9545
@26;949576;:824687008128
