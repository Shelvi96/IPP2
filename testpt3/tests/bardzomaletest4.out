643092859
3
6840835047105822720
0
0
31001
0
6840835047105822720
817887699
0
0
6840835047105822720
0
0
0
80172168879
